packages/integration/api-report.md

```diff
@@ -539,7 +552,7 @@ export type GiteaIntegrationConfig = {
   host: string;
   baseUrl?: string;
   username?: string;
-  password?: string;
+  token?: string;
```

Comment:
Let's not change anything in Gittea configs :)

Reply:
fixed